File: C/easy-medium/power calculation.c
#include <stdio.h>
#include <math.h>

int main() {
    float base, result;
    int exponent;
    printf("Enter base: ");
    scanf("%f", &base);
    printf("Enter exponent: ");
    scanf("%d", &exponent);
    result = pow(base, exponent);
    printf("Result: %.2f\n", result);
    return 0;
}

File: JAVA/easy-medium/power calculation.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter base: ");
        double base = scanner.nextDouble();
        System.out.print("Enter exponent: ");
        int exponent = scanner.nextInt();
        double result = Math.pow(base, exponent);
        System.out.println("Result: " + result);
    }
}
